srcCxx/shape_main.cxx

```diff
@@ -1354,25 +1343,7 @@ class ShapeApplication {
 
         logger.log_message("Subscriber QoS:", Verbosity::DEBUG);
 
-#if   defined(RTI_CONNEXT_DDS)
-        if (options->coherent_set_enabled) {
-            sub_qos.presentation.coherent_access = DDS_BOOLEAN_TRUE;
-        }
-        if (options->ordered_access_enabled) {
-            sub_qos.presentation.ordered_access = DDS_BOOLEAN_TRUE;
-        }
-        if (options->ordered_access_enabled || options->coherent_set_enabled) {
-            sub_qos.presentation.access_scope = options->coherent_set_access_scope;
-        }
-
-        logger.log_message("    Presentation Coherent Access = " +
-                std::string(sub_qos.presentation.coherent_access ? "true" : "false"), Verbosity::DEBUG);
-        logger.log_message("    Presentation Ordered Access = " +
-                std::string(sub_qos.presentation.ordered_access ? "true" : "false"), Verbosity::DEBUG);
-        logger.log_message("    Presentation Access Scope = " +
-                QosUtils::to_string(sub_qos.presentation.access_scope), Verbosity::DEBUG);
-
-#elif defined(TWINOAKS_COREDX)
+#if   defined(RTI_CONNEXT_DDS) || defined(TWINOAKS_COREDX)
```

Comment:
@ornulfs @ornulfs-kda I think that here you need to add the Intercom flag as well